xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -227,18 +227,29 @@ def safe_exec(
                     "extra_files": extra_files,
                 }
                 with function_trace('safe_exec.remote_exec_darklaunch'):
-                    remote_emsg, _remote_exception = get_remote_exec(data)
+                    remote_emsg, _ = get_remote_exec(data)
+            except BaseException as e:  # pragma: no cover  # pylint: disable=broad-except
+                # Swallow all exceptions and log it in monitoring so that dark launch doesn't cause issues during
+                # deploy.
+                remote_exception = e
```

Comment:
Don't we want to explicitly set `remote_emsg` here to something?

Reply:
....yes, and I should probably push my local changes. :facepalm:

Reply:
That's fixed, and I've added some tests, one of which catches the behavior I'm fixing here.

There's another bug to fix (around globals) but I will likely handle that as a separate PR.

Reply:
OK, I went ahead and added a fix for the globals issue as well (see most recent commit).